doc/source/api.rst

```diff
@@ -5,6 +5,21 @@
 API Reference
 *************
 
+This page gives an overview of all public pandas objects, functions and
+methods. In general, all classes and functions exposed in the top-level
+``pandas.*`` namespace are regarded as public.
+Further some of the subpackages are public as well, including `pandas.errors``,
```

Comment:
Missing an opening backtick.

And could these all be be links?

    :mod:`pandas.errors`

Though I don't think we do any module-level things in the toctree, so maybe not.